src/routes/ErrorPage.tsx

```diff
@@ -16,10 +17,13 @@ export default function ErrorPage(): ReactElement {
 
   if (isErrorResponse(error)) {
     errorMessage = error.status + " " + error.statusText;
+    triggerAnalyticsEvent(AnalyticsEvent.ROUTE_ERROR, { errorMessage: error.statusText, errorStatus: error.status });
   } else if (error instanceof Error) {
     errorMessage = error.message;
+    triggerAnalyticsEvent(AnalyticsEvent.ROUTE_ERROR, { errorMessage: error.message });
   } else {
     errorMessage = "Unknown error";
+    triggerAnalyticsEvent(AnalyticsEvent.ROUTE_ERROR, { errorMessage: "Unknown error" });
```

Comment:
if someone mistakenly copy/pastes a bad data set url will one of these analytics errors happen? Any concern about a disproportionate amount of these coming through?

Reply:
Routing errors are catastrophic errors that usually indicate a bug in the viewer code rather than an issue with the dataset. Dataset errors *should* be shown on the UI via a popup (and not trigger the `ROUTE_ERROR` event), though there's almost certainly some edge cases we haven't caught yet from weird/nonsensical dataset configurations.